CIScripts/Tests/TestConfigurationUtils.ps1

```diff
@@ -292,10 +300,11 @@ function Initialize-TestConfiguration {
     $SleepTimeBetweenChecks = 10;
     $MaxNumberOfChecks = $WaitForSeconds / $SleepTimeBetweenChecks
 
-    # Wait for vRouter Extension to be started by DockerDriver
+    # Wait for DockerDriver to listen.
+    # (Docker driver should listen only after enabling extension)
     $Res = $false
     for ($RetryNum = $MaxNumberOfChecks; $RetryNum -gt 0; $RetryNum--) {
```

Comment:
FYI this loop can also be written as
```
1..10 | % { echo "asdf" }
```
(I really like the `$a..$b` syntax, just wanted to share :smile: )

Reply:
Should I change it?

Reply:
I mean I'm not gonna stop you, but I'm not requesting a change here, just sharing syntax ;)